script.js

```diff
@@ -1,52 +1,68 @@
 const formBtn = document.querySelector('#form-button');
 const booksList = document.querySelector('.books-list');
+let books = JSON.parse(localStorage.getItem('books'));
 
-let books = [];
+class Book {
+  constructor(id, title, author) {
+    this.id = id;
+    this.title = title;
+    this.author = author;
+  }
 
-const booksAdd = (id, title, author) => {
-  const li = document.createElement('li');
-  li.innerHTML = `<h2>${title}</h2>
-    <h3>${author}</h3>
-    <hr>`;
-  const deleteBtn = document.createElement('button');
-  deleteBtn.textContent = 'Remove';
-  li.insertBefore(deleteBtn, li.lastElementChild);
-  booksList.appendChild(li);
-  deleteBtn.addEventListener('click', () => {
-    books = books.filter((books) => {
-      if (books.title !== title && books.author !== author) {
+  addBook() {
+    const { title, author } = this;
+    const bookObj = { title, author };
+    if (title !== '' || author !== '') {
+      books.push(bookObj);
```

Comment:
- [ ] OPTIONAL
It's a good idea to use non mutating array methods `concat`, over `push`, so as to keep the original array intact.

Reply:
Thanks for the review.